recipe/meta.yaml

```diff
@@ -31,24 +30,25 @@ build:
 requirements:
     build:
         - {{ compiler('c') }}
+        - {{ compiler('cxx') }}
         - {{ compiler('dpcpp') }}
         - patch     # [not win]
         - m2-patch  # [win]
         - cmake >=3.21
-        - git
+        - git  # [not win]
         - ninja
         - cython
     host:
         - setuptools
         - cython
-        - pybind11 2.9.2
+        - pybind11 2.10.4
         - python
         - scikit-build
-        - numpy 1.21
+        - numpy 1.22
         - wheel
```

Comment:
I might be wrong here, but I thought that wheel is mainly used in conjunction with pip, i.e. "python -m pip install". In this recipe, 'python setup.py install' is used. Do we need wheel? Or, should we build using pip instead?